src/Illuminate/Foundation/Testing/Concerns/InteractsWithViews.php

```diff
@@ -33,13 +34,13 @@ protected function blade(string $template, array $data = [])
     {
         $tempDirectory = sys_get_temp_dir();
 
-        if (! in_array($tempDirectory, ViewFacade::getFinder()->getPaths())) {
+        if (!in_array($tempDirectory, ViewFacade::getFinder()->getPaths())) {
```

Comment:
please revert unrelated changes

Reply:
ahhh... my editor fixed the styling when I save...